### PR TITLE
Show autoload changes in readme for upgrade beta3 -> 1.0 (fixes #8564)

### LINKED ISSUE #8564: Upgrade Feedback: Mention autoload changes in upgrade guide
https://github.com/Sylius/Sylius/pull/8340/files

I think this should be mentioned in the upgrade guide from beta-3 -> 1.0.

### PR DESCRIPTION
**Pull request by stefandoorn**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no|
| BC breaks?      | no|
| Related tickets | fixes #8564 |
| License         | MIT |

### CODE REVIEW

**Comment by lchrusciel** [comment]
The only thing that disturbs me is how does it work for the Symfony versions below 3.3.8?

**Comment by stefandoorn** [comment]
Good one, I actually have no clue. This is the commit in Symfony-Standard. I interpreted it as that it's only included in 3.3.8: https://github.com/symfony/symfony-standard/commit/298f8b26f3f7ad834a918a6970822ba37699b885.

So that means that before 3.3.8, the app/autoload would still exist and I guess required. Maybe that's also good to note in the Upgrade guide then. 

Or, should we require Symfony ^3.3.8 instead of ^3.3.6?

**Comment by stefandoorn** [comment]
But that's 3.3.8 from Symfony-Standard. I think this is the one related that's in 3.3: https://github.com/symfony/symfony/pull/21837.

**Comment by stefandoorn** [comment]
@lchrusciel Adjusted text to make users only do this when on Symfony ^3.3.8.

**Comment by lchrusciel** [comment]
I suppose that current changes force us to use symfony 3.3.8 anyway, so maybe the comment won't be needed. Let's wait until tomorrow for others /cc @Sylius/core-team

File: UPGRADE.md [hunk 1]
```diff
@@ -24,6 +24,17 @@
 * There are scalar and return typehints introduced in the codebase. Please introduce these in your codebase for classes 
   that implement Sylius' interfaces or extend Sylius' classes.
 
+* **Only when on Symfony ^3.3.8:**
+  From this Symfony version, the custom autoloader is not needed anymore and therefore removed in favor of the Composer 
+  autoloader. Apply the following changes (reference: https://github.com/Sylius/Sylius/pull/8340):
+
+  * Remove `app/autoload.php`
+  * Change autoload path in `bin/console`: replace 'vendor' with 'app'
```
Comment (pjedrzejewski) [inline UPGRADE.md]: I think you meant the other way around? :)

Comment (stefandoorn) [reply UPGRADE.md]: Wow, haha, indeed :)

**Comment by lchrusciel** [comment]
Closing in favour of #8576

**Comment by lchrusciel** [comment]
Sorry, not this one :)

**Comment by stefandoorn** [comment]
Feedback fixed.

**Comment by pjedrzejewski** [comment]
Thank you Stefan! 👍

**Pull request merged**